src/main/java/io/blamer/bot/request/RegistryRequest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2023 Blamer.io
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.blamer.bot.request;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * Request for auth.
+ */
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class RegistryRequest {
```

Comment:
@l3r8yJ maybe `RqRegistry`?

WDYT?

Reply:
@h1alexbel We have no interfaces to add these prefixes `Rq`/`Rs`. I think prefixes should be like markers

Reply:
@l3r8yJ maybe we should implement an interface `Registry`? > RqRegistry will be concrete class

Reply:
@h1alexbel i think better would be implement `Request`/`Response` for `Rq`/`Rs`, wdyt?

Reply:
@h1alexbel but I suggest this refactoring when we need additional responses/requests

Reply:
@l3r8yJ agree with you